components/shared/AppSidebar/AppSidebar.tsx

```diff
@@ -92,204 +92,209 @@ export const AppSideBar = ({ workspaceId, hideSidebar, sidebarCollapsed }: AppSi
   return (
     // TODO: get rid of the z-index. There is grid content like the avatars and paged data text that bleed through the sidebar atm.
     <div
-      className={`fixed left-0 w-72 bg-white shadow-lg transform  transition-transform duration-300 ease-in-out border-r ${
-        sidebarCollapsed ? "" : "-translate-x-full"
+      aria-hidden={sidebarCollapsed}
+      className={`fixed left-0 w-72 bg-white shadow-lg transform transition-transform duration-300 ease-in-out border-r ${
```

Comment:
with the fix on the `sidebarCollapsed` prop, some of the logic is reversed now, i.e.

```diff
- sidebarCollapsed ? "" : "-translate-x-full"
+ sidebarCollapsed ? "-translate-x-full" : ""
```